src/components/modal/Modal.scss

```diff
@@ -144,6 +144,10 @@
             padding: 0;
         }
 
+        .modal-content {
+            padding-bottom: 60px;
```

Comment:
@msorianoBox Sorry about this! I think my math/process was incomplete and it should be `78px`?

The buttons are 40px with 10px on the top and 10px on the bottom = 60px total. So the height of `modal-actions` that's located at the bottom of the screen is 60px.

But since we have `margin-top: 30px` on `modal-actions` in the normal view (the non-responsive view and I assume we want to include this similar margin/design), the end of the content would be `90px` from the bottom of the screen.

We would add this padding to `modal-content` so that there's no content behind the `modal-actions` bar and there's enough white space between the buttons and the content. But we also need to take into account the bottom padding that's already included for `modal-dialog`. So that would be 90px minus 12px = 78px.

Would love a second opinion on this though. It's also very possible that I'm overthinking this :sweat_smile:.

Reply:
Also happy with just leaving this as-is.

Reply:
In fact, I was testing with some overflowing content and 40 px worked perfectly to display all of the content without cropping. I went for the 60 px, since you requested and even with the extra white space looked good. I really think that there's no need for anymore extra margin, but if you want, I'll add it :D